io_scene_nif/modules/nif_export/property/object/__init__.py

```diff
@@ -123,10 +128,54 @@ def get_matching_block(self, block_type, **kwargs):
                 setattr(block, param, attribute)
         return block
 
-    # TODO [material][property] Move this to new form property processing
+    def export_properties(self, b_obj, b_mat, n_block):
```

Comment:
I would pull this up higher in the class as it looks to be the main processing method.

Reply:
Right it is sort of a wrapper for all property exporters